### PR TITLE
how to get a meter and tracer with agent and spring starter

### PR DESCRIPTION
**Pull request by zeitlinger**

### CODE REVIEW

File: content/en/docs/languages/java/instrumentation.md [hunk 1]
```diff
@@ -678,7 +678,8 @@ brittle option for library instrumentation.
 
 ### Acquiring a Tracer in Java Agent
 
-If you are writing a Java Agent, you can acquire a `Tracer` from the global OpenTelemetry instance:
+If you are writing a Java Agent, you can acquire a `Tracer` from the global
```
Comment (trask) [inline content/en/docs/languages/java/instrumentation.md:681]:
```suggestion
If you are using the OpenTelemetry Java Agent, you can acquire a `Tracer` from the global
```

File: content/en/docs/languages/java/instrumentation.md [hunk 2]
```diff
@@ -688,7 +689,8 @@ Tracer tracer = GlobalOpenTelemetry.getTracer("instrumentation-scope-name");
 
 ### Acquiring a Tracer in Spring Boot starter
 
-If you are writing a Spring Boot starter, you can acquire a `Tracer` from the autowired OpenTelemetry instance:
+If you are writing a Spring Boot starter, you can acquire a `Tracer` from the
```
Comment (trask) [inline content/en/docs/languages/java/instrumentation.md:692]:
```suggestion
If you are using the OpenTelemetry Spring Boot Starter, you can acquire a `Tracer` from the
```

File: content/en/docs/languages/java/instrumentation.md [hunk 3]
```diff
@@ -1245,7 +1247,8 @@ you can create
 
 ### Acquiring a Meter in Java Agent
 
-If you are writing a Java Agent, you can acquire a `Meter` from the global OpenTelemetry instance:
+If you are writing a Java Agent, you can acquire a `Meter` from the global
```
Comment (trask) [inline content/en/docs/languages/java/instrumentation.md:1250]:
```suggestion
If you are using the OpenTelemetry Java Agent, you can acquire a `Meter` from the global
```

File: content/en/docs/languages/java/instrumentation.md [hunk 4]
```diff
@@ -1255,7 +1258,8 @@ Meter meter = GlobalOpenTelemetry.getMeter("instrumentation-scope-name");
 
 ### Acquiring a Meter in Spring Boot starter
 
-If you are writing a Spring Boot starter, you can acquire a `Meter` from the autowired OpenTelemetry instance:
+If you are writing a Spring Boot starter, you can acquire a `Meter` from the
```
Comment (trask) [inline content/en/docs/languages/java/instrumentation.md:1261]:
```suggestion
If you are using the OpenTelemetry Spring Boot Starter, you can acquire a `Meter` from the
```

**Review by trask — COMMENTED** [review]
(maybe also links to the Java agent and Spring Boot Starter pages?)

**Review by svrnm — APPROVED** [review]

File: content/en/docs/languages/java/instrumentation.md [hunk 5]
```diff
@@ -676,6 +676,36 @@ Tracer tracer = GlobalOpenTelemetry.getTracer("instrumentation-scope-name", "ins
 Note that you can't force end users to configure the global, so this is the most
 brittle option for library instrumentation.
 
+### Acquiring a Tracer in Java agent
```
Comment (theletterf) [inline content/en/docs/languages/java/instrumentation.md:679]:
```suggestion
### Acquiring a tracer in Java agent
```

Sentence case in titles unless there's a proper noun, such as "Spring Boot".

**Review by theletterf — APPROVED** [review]

File: content/en/docs/languages/java/instrumentation.md (hunk 5 above)
Comment (zeitlinger) [reply content/en/docs/languages/java/instrumentation.md]: also changed the existing headers

**Review by trask — APPROVED** [review]

**Pull request merged**